lib/core/Stage.cpp

```diff
@@ -7,18 +7,20 @@
 
 #include "fmt.hpp" // for format
 
-#include <algorithm>    // for copy, max
-#include <chrono>       // for seconds
-#include <cstdlib>      // for abort
-#include <cxxabi.h>     // for __forced_unwind
-#include <exception>    // for exception
-#include <future>       // for async, future, future_status, future_status::timeout, launch
-#include <pthread.h>    // for pthread_setaffinity_np, pthread_setname_np
-#include <regex>        // for match_results<>::_Base_type
-#include <sched.h>      // for cpu_set_t, CPU_SET, CPU_ZERO
-#include <stdexcept>    // for runtime_error
-#include <system_error> // for system_error
-#include <thread>       // for thread
+#include <algorithm>     // for copy, max, find
+#include <chrono>        // for seconds
+#include <cstdlib>       // for abort
+#include <cxxabi.h>      // for __forced_unwind
+#include <exception>     // for exception
+#include <future>        // for async, future, future_status, future_status::timeout, launch
+#include <pthread.h>     // for pthread_setaffinity_np, pthread_setname_np
+#include <regex>         // for match_results<>::_Base_type
+#include <sched.h>       // for cpu_set_t, CPU_SET, CPU_ZERO
+#include <stdexcept>     // for runtime_error
+#include <sys/syscall.h> // for SYS_gettid
```

Comment:
```suggestion
#include <syscall.h> // for SYS_gettid
```
I think this will fix the iwyu error.

Reply:
This will cause another test failure unfortunately. it cannot find this file without sys/

Reply:
Ah that's annoying.  We should just tell IWYU to ignore this include problem then: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUPragmas.md